### PR TITLE
instant-messenger: add warning to Signal [Discussion]

### PR DESCRIPTION
**Pull request by ggg27**
**Description**: Elaborate more for Signals Description + Warning
**What is wrong with Signal?**: Signal depends on proprietary software for its Android application. Therefore, is not "[free software](https://www.wikipedia.org/wiki/Free_software_movement)" as listed before.
**Why does freedom matter?**: Users cannot fully audit Signal's Android application. This is particularly worrisome if it plans to replace modern SMS. 
More Info: https://www.gnu.org/philosophy/right-to-read.html

### CODE REVIEW

File: _includes/sections/instant-messenger.html [hunk 1]
```diff
@@ -8,8 +8,8 @@ <h1 id="im" class="anchor"><a href="#im"><i class="fas fa-link anchor-icon"></i>
 {% include cardv2.html
 title="Mobile: Signal"
 image="/assets/img/tools/Signal.png"
-description="Signal is a mobile app developed by Open Whisper Systems. The app provides instant messaging, as well as voice and video calling.
-All communications are end-to-end encrypted. Signal is free and open source."
+description='Signal is a mobile app developed by Open Whisper Systems. The app provides instant messaging, as well as voice and video calling.
+All communications are end-to-end encrypted. <span class="badge badge-warning" data-toggle="tooltip" title="Signal has non-free dependecies for Android. Is it strongly un-advised to use on Android until fixed.">warning <i class="far fa-question-circle"></i> (<a href="https://github.com/signalapp/Signal-Android/issues/8927">more info</a>)'
```
Comment (Mikaela) [inline _includes/sections/instant-messenger.html]: These "more details" at https://github.com/signalapp/Signal-Android/issues/8927 tell me nothing about the issue. There is an archive.is link to article from Jan 13, 2017 and a link to https://github.com/signalapp/Signal-Android/issues/7988 which tells nothing either. If you follow two links more you end up to https://github.com/signalapp/Signal-Android/issues/127#issuecomment-13335689 where the author says to have no interest in F-Droid redistribution and I think F-Droid wants permission from the author for that.

Comment (Mikaela) [reply _includes/sections/instant-messenger.html]: At Matrix I was pointed out that you also have a typo, dependecies should be dependencies.

Comment (ggg27) [reply _includes/sections/instant-messenger.html]: Thanks, the typo should be fixed now.

Comment (ggg27) [reply _includes/sections/instant-messenger.html]: > These "more details" at signalapp/Signal-Android#8927 tell me nothing about the issue. 

Thanks for the feedback, I agree and changed the link to:
https://forum.privacytools.io/t/signal-has-non-free-dependencies-android/1115

Which I hope to provide more information. Thanks for pointing this out.

**Comment by Mikaela** [comment]
> Thanks for the feedback, I agree and changed the link to:
https://forum.privacytools.io/t/signal-has-non-free-dependencies-android/1115
> 
> Which I hope to provide more information. Thanks for pointing this out!

There seems to be concensus that the information is outdated, please request reopening if you can confirm it to not be the case.

**Pull request closed without merging**